libr/core/canal.c

```diff
@@ -1926,12 +1926,29 @@ R_API int r_core_anal_esil_fcn(RCore *core, ut64 at, ut64 from, int reftype, int
 	return 0;
 }
 
+static int find_sym_flag(void *a1, void *a2) {
+	RFlagItem *f = (RFlagItem *)a2;
+	return f->space && !strcmp (f->space->name, R_FLAGS_FS_SYMBOLS)? 0: 1;
+}
+
+static bool is_skippable_addr(RCore *core, ut64 addr) {
+	RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, addr, 0);
+	if (!fcn) {
+		return false;
+	}
+	if (fcn->addr == addr) {
+		return true;
+	}
+	const RList *flags = r_flag_get_list (core->flags, addr);
```

Comment:
what about using rbin instead of flags to determine if there's a symbol? cc @ret2libc @pelijah ?

Reply:
I'm not sure how the change would affect the analysis, it should be tried.

Reply:
I suggest we merge this as is, but consider this change in a new PR.